Documentation/ApiOverview/Database/BasicCrud/_MyUpdateRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor\MyExtension\Domain\Repository;
+
+use TYPO3\CMS\Core\Database\ConnectionPool;
+
+class MyUpdateRepository
+{
+    private ConnectionPool $connectionPool;
+
+    public function __construct(ConnectionPool $connectionPool)
+    {
+        $this->connectionPool = $connectionPool;
+    }
+
+    public function updateSomeData()
+    {
+        $this->connectionPool->getConnectionForTable('tt_content')
+            ->update(
+                'tt_content',
+                [ 'bodytext' => 'bernd' ], // set
```

Comment:
Let us use more international names?

Reply:
Thought also about it. But these examples, "bernd, "klaus", "peter" are found throughout the database section. I have them a lot in the query builder chapter. We can do it, but after all the streamlining is done in a separate patch.

Reply:
ack